deflate.c

```diff
@@ -296,11 +296,10 @@ int ZEXPORT deflateInit2_(strm, level, method, windowBits, memLevel, strategy,
     }
 #endif
     if (memLevel < 1 || memLevel > MAX_MEM_LEVEL || method != Z_DEFLATED ||
-        windowBits < 8 || windowBits > 15 || level < 0 || level > 9 ||
```

Comment:
I'm probably missing something but I thought that bug was just in `windowBits=8`. Does fixing the 256-byte window bug mean all values of `windowBits` from 0 to 15 are now valid?